config/locales/blacklight.zh.yml

```diff
@@ -0,0 +1,212 @@
+en:
```

Comment:
Thanks @yuxisteem -- this looks great, and just needs one small change before we merge. The `en` here on line 1 should be `zh` to match the locale.

Reply:
Sorry for this, I've got it changed.